plugins/affiliation/src/main.rs

```diff
@@ -322,64 +355,56 @@ async fn affiliation(engine: &mut PluginEngine, key: Target) -> Result<i64> {
 
 	log::info!("completed affiliation metric");
 
-	Ok(count)
+	Ok(affiliations)
```

Comment:
It's hard to tell with the logic, but it doesn't appear that this function returns 1 bool per commit the way that its header comment says it does. Can you verify that `affiliations.len() == hashes.len()`?

Reply:
That would be because I forgot to update the header. Affiliations should have 1 bool per contributor, not commit. Testing confirms that. Will update the header comment.